### DISCUSSION TITLE
How to handle promises in deactivate?

### DISCUSSION THREAD

**Discussion by sudoaugustin** [microsoft/vscode-discussions#41] [Extension Development QnA] [answered]
I want my vscode not to close until my promise is finished.

**Comment by sudoaugustin** [comment, marked as answer]
I just found another discussion about the same issue I'm facing and would like to close or delete this discussion.

**Reply by sudoaugustin** [reply]
Sry for the trouble making

**Reply by digitarald** [reply]
Thanks for doing the investigations. Maybe you can link the other discussion here?
